e3sm_diags/derivations/acme.py

```diff
@@ -2074,3 +2074,391 @@ def cosp_histogram_standardize(cld: "FileVariable"):
     "sitemptop": OrderedDict([(("sitemptop",), rename)]),
     "siv": OrderedDict([(("siv",), rename)]),
 }
+
+# Names of 2D aerosol burdens, including cloud-borne aerosols
+aero_burden_list = [
+    "ABURDENDUST",
+    "ABURDENSO4",
+    "ABURDENSO4_STR",
+    "ABURDENSO4_TRO",
+    "ABURDENPOM",
+    "ABURDENMOM",
+    "ABURDENSOA",
+    "ABURDENBC",
+    "ABURDENSEASALT",
+]
+
+
+def aero_burden_fxn(var):
+    """
+    Scale the aerosol burden by 1e6.
+
+    Parameters:
+        var (float): The input burden in kg/m2.
+
+    Returns:
+        burden (float): The output burden in 1e-6 kg/m2.
+    """
+    burden = var * 1e6
+    burden.units = "1e-6 kg/m2"
+    return burden
+
+
+# Add burden vars to derived_variables
+for aero_burden_item in aero_burden_list:
+    derived_variables[aero_burden_item] = OrderedDict(
+        [((aero_burden_item,), aero_burden_fxn)]
+    )
+
+
+# Names of 2D mass slices of aerosol species
+# Also add 3D masses while at it (if available)
```

Comment:
I can't seem to find codes to support 3D masses..

Reply:
hey @mahf708, I should have be clearer, for 3D mass, I was going to suggest to have the function to [create column integrated](# https://github.com/E3SM-Project/e3sm_diags/blob/633b52c314325e605fe7f62687cc4d00e5a0a3d5/e3sm_diags/driver/aerosol_budget_driver.py#L29 ) value from 3D aerosol mass concentration in derived variable list under the entries of ABURDENXX, the entry in derived_variable would be something like.
```
"ABURDENBC": OrderedDict(
        [
            (("ABURDENBC",), aero_burden_fxn),
            # If ABURDENBC is not exist, detect if Mass_bc is available, and do column integral.
            (("Mass_bc","hyai", "hybi", "PS"), calc_column_integral),              
        ]
    ), 
```
This could be helpful when re-run e3sm_diags with simulations prior to v3.

Reply:
Sounds good. Other changes in this PR looks good to me!